tools/cli/src/main.rs

```diff
@@ -89,20 +91,24 @@ fn aqua(args: &clap::ArgMatches<'_>) -> Result<(), anyhow::Error> {
         println!("{}", record);
     }
 
-    match args.value_of(args::SERVICE_NAME) {
-        Some(service_name) => println!("service {}:", service_name.to_title_case()),
```

Comment:
What was bad with `.to_title_case()`?

Reply:
I got it, but maybe it's better to find a crate for that?

Reply:
Or, probably, [to_class_case](https://docs.rs/Inflector/0.11.4/inflector/cases/classcase/index.html) is what we need here

Reply:
sounds like a solution!